snyk/snyk
# feat: composer dev require

#### What does this PR do?

Bump snyk-php-plugin to v1.7.0.
This version includes facility to parse dev packages.
Added acceptance test to ensure dev option would be passed to
snyk-php-plugin.
Added dev dependency to composer-app test fixture.

#### How should this be manually tested?

Run CLI against composer project with '--dev' flag, to see results make sure the project has require-de dependencies.

#### What are the relevant tickets?

BST-917

#### Screenshots

Example using composer-with-vulns snyk-fixture:
![Screen Recording 2019-11-06 at 12 05 50 2019-11-06 12_11_22](https://user-images.githubusercontent.com/6598617/68297463-e2e16600-008e-11ea-9f79-76ec5a6c376a.gif)
